modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
 . Save the `.bashrc` file.
 . Restart your terminal.
 
+== Preparing to Install Instio
```

Comment:
```suggestion
== Preparing to Install Istio
```

Reply:
Be sure to fix this GLOBALLY. It looks like your reviewer has caught them all, but double-check after you update the PR.
Also, please check for and resolve reviewer comments BEFORE submitting PRs for editing. Thanks!

Reply:
After reading further, I see that this topic is NOT about PREPARING to install; it is about installing different versions of Instio. Please clarify.

Reply:
Done! Thanks!